sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -196,7 +196,7 @@ statement
     | SHOW PARTITIONS multipartIdentifier partitionSpec?               #showPartitions
     | SHOW identifier? FUNCTIONS
         (LIKE? (multipartIdentifier | pattern=STRING))?                #showFunctions
-    | SHOW CREATE TABLE multipartIdentifier                            #showCreateTable
+    | SHOW CREATE TABLE multipartIdentifier (AS SPARK)?                #showCreateTable
```

Comment:
After rethinking it, let us make it more aggressive here. Instead of creating Spark native tables for the existing Hive serde tables, we can try to always show how to create Spark native tables if possible. This will further simplify the migration from Hive to Spark.

To the existing Spark users who prefer to keeping Hive serde formats, we can introduce a new option `AS SERDE` which will keep the behaviors in Spark 2.4 or prior.

Reply:
+1. The new proposal makes more sense!

Reply:
A bit confusing and let me confirm. So you mean let SHOW CREATE TABLE work with `AS SPARK` by default, and only fallback to current behavior (show how to create Hive serde table) when given `AS SERDE`?

Reply:
Yes!